bingosync-app/generators/dark_souls_generator.js

```diff
@@ -76,12 +74,11 @@ bingoList[7] = [
   { name: "Obtain Whip", types: ["weapons","blighttown"] },
   { name: "Obtain Server", types: ["weapons","blighttown"] },
   { name: "Obtain Iaito", types: ["weapons","blighttown"] },
-  { name: "Obtain Force Within", types: ["items","spells","blighttown"] },
```

Comment:
There's no item called **Force Within**! This could refer to **Force** or **Power Within**, but since it's in the Blighttown array **Power Within** was almost certainly intended.